### ISSUE TITLE
Missing compatibility with 3.2 / 3.3.

### ISSUE THREAD

**Issue by anselmdk** [sheadawson/silverstripe-blocks#44]
I'm getting the following when trying to install:

```
sheadawson/silverstripe-blocks dev-master requires silverstripe/cms 3.1.* -> satisfiable by silverstripe/cms[3.1.x-dev].
```

Are there any blockers that prevent the module from being compatible with 3.2/3.3 ?

**Comment by sheadawson** [comment]
Just did a quick test in 3.2, all seems happy. Released 1.0.4 with ~3.1 dependency for cms :)

**Comment by anselmdk** [comment]
Great, thanks. I just upgraded to `3.3.x-dev`, and it seems to be working fine. It's now that behat tests would come in handy :)
